js/controllers/grid/navigationMenus/form/NavigationMenuItemsFormHandler.js

```diff
@@ -112,18 +111,10 @@
 	$.pkp.controllers.grid.navigationMenus.form.NavigationMenuItemsFormHandler.
 			prototype.setType = function() {
 		var itemType = $('#menuItemType', this.getHtmlElement()).val(),
-				$customPageEls = $('#customPageOptions, #previewButton'),
-				$remoteUrlEls = $('#remoteUrlTarget'),
 				$descriptionEl = $('#menuItemTypeSection [for="menuItemType"]');
 
-		$customPageEls.hide();
-		$remoteUrlEls.hide();
-
-		if (itemType === 'NMI_TYPE_CUSTOM') {
-			$customPageEls.fadeIn();
-		} else if (itemType === 'NMI_TYPE_REMOTE_URL') {
-			$remoteUrlEls.fadeIn();
-		}
+		$(".NMI_TYPE_CUSTOM_EDIT").hide();
```

Comment:
When using a jQuery selector that isn't a specific ID, add `this.getHtmlElement` as the second argument. It will restrict jQuery's element search to just elements inside the handler. That will make it faster and also less likely to conflict with another element on the page.

```
$('.anySelector', this.getHtmlElement());
```